src/common/src/system_param/common.rs

```diff
@@ -33,7 +35,11 @@ impl CommonHandler {
     /// Handle the change of system parameters.
     pub fn handle_change(&self, diff: &SystemParamsDiff) {
         if let Some(enabled) = diff.enable_tracing {
-            toggle_otel_layer(enabled)
+            toggle_otel_layer(enabled);
+        }
+        if let Some(key) = diff.license_key.as_ref() {
+            LicenseManager::get().refresh(key);
```

Comment:
Will this code be called for Meta node itself? This does matter if a feature check is performed on Meta node.

Reply:
Yes. All nodes will have this handler run.